pkgs/tools/misc/pridefetch/default.nix

```
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchFromGitHub, python3, zip }: let
+  version = "1.0.0";
+  sha256 = "sha256-/o4er8bO/3HUFXzP+sC+5DYv9EwmxW05o1RT5fEulEg=";
+
+  pname = "pridefetch";
+  src = fetchFromGitHub {
+    owner = "SpyHoodle";
+    repo = pname;
+    rev = "v" + version;
+    inherit sha256;
+  };
+in stdenv.mkDerivation {
+  inherit pname version src;
+  nativeBuildInputs = [
+    zip
+  ];
+  buildInputs = [
+    (python3.withPackages (pythonPackages: with pythonPackages; [
+      distro
+    ]))
+  ];
+  buildPhase = ''
+    runHook preBuild
+    builddir=$(pwd)
+    pushd src
+    zip -r $builddir/pridefetch.zip ./*
```

```suggestion
    pushd src
    zip -r ../pridefetch.zip .
```

Relative paths are fine here but in general, prefer `$PWD` to `$(pwd)`.